### ISSUE TITLE
DynamicAperture Softening parameter is dependent on the pixel scale

### ISSUE THREAD

**Issue by LouisDesdoigts** [LouisDesdoigts/dLux#224]
The softening parameter is meant to be agnostic to the physical scale of the aperture itself, but this is not the case. 

Example:

```python
import dLux as dl
import matplotlib.pyplot as plt

radius = 1e-6
aper = dl.CircularAperture(1e-6)
plt.imshow(aper.get_transmission(256, 2*radius))
```
<img width="517" alt="Screenshot 2023-04-20 at 1 50 19 pm" src="https://user-images.githubusercontent.com/42196907/233253583-5257a343-930a-4602-8734-b2a0910dd417.png">

We need to set softening to a very small value to get sensible results

```python
aper = dl.CircularAperture(1e-6, softening=1e-5)
plt.imshow(aper.get_transmission(256, 2*radius))
```
<img width="517" alt="Screenshot 2023-04-20 at 1 51 21 pm" src="https://user-images.githubusercontent.com/42196907/233253723-69b4151d-8644-42d2-b328-efe286b278bc.png">

This will be an issue for coronagraphs that have small angular pixel scales in the occulter plane.

---

**Comment by LouisDesdoigts** [comment]
@Jordan-Dennis would love your thoughts. I briefly tried to add a `scale` parameter to the `_soften` method taken from the coordinates but it seems the solution will be a little more involved than that.

**Comment by LouisDesdoigts** [comment]
Fixed with #246
